### PR TITLE
Add section 3.1.8 handling interlinear punctuation

### LINKED ISSUE #463: 需要增加关于行间标点排版的内容
目前文档只阐述了行内标点的问题，但对于着重号、专名号等行间标点的需求，阐述甚少，需要补充

### PR DESCRIPTION
**Pull request by huijing**
Closes https://github.com/w3c/clreq/issues/463

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/pull/489.html" title="Last updated on Nov 29, 2022, 11:55 AM UTC (bfb9d3f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/489/8ccc302...bfb9d3f.html" title="Last updated on Nov 29, 2022, 11:55 AM UTC (bfb9d3f)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2078,7 +2078,87 @@ <h4>
...
+      </h4>
+
+      <p its-locale-filter-list="en" lang="en">Most punctuation marks in modern Chinese typesetting are interspersed between the text in a line, but the Proper Noun Mark (underline), the Book Title Mark (only for the type A wavy low line) and the Emphasis Dot need to be positioned between lines. These punctuation marks are known as “interlinear punctuation”. Of which, the Proper Noun Mark and the Book Title mark are known as interlinear line marks.</p>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">现代中文排版的标点符号，大部分都随文字穿插在行内，但专名号（下划线）、书名号（仅指甲式的波浪线)、着重号要摆放在行与行之间，这些标点通称为「行间标点」。其中专名号（下划线）、书名号（仅指甲式的波浪线）合称为「行间线」。</p>
+      <p its-locale-filter-list="zh-hant" lang="zh-hant">現代中文排版的標點符號，大部分都隨文字穿插在行內，但專名號（下劃線）、書名號（僅指甲式的波浪線)、著重號要擺放在行與行之間，這些標點通稱為「行間標點」。其中專名號（下劃線）、書名號（僅指甲式的波浪線）合稱為「行間線」。</p>
+
+      <p its-locale-filter-list="en" lang="en">Sentence spacing for ancient Chinese texts were all interlinear punctuation. Although the typesetting of ancient Chinese texts is beyond the scope of this document, the guidance provided in this section can still serve as a reference for the basic principles of interlinear punctuation.</p>
```
Comment (xfq) [inline index.html]: "Sentence spacing" means the spaces inserted between sentences, but IIUC 句读 is a different thing...

**Comment by xfq** [comment]
I discussed this PR with @r12a offline. The following is a summary. He can explain in more detail.

1. Interlinear line marks (行间线) refer to the line between two lines of text, but it is not necessarily between the two lines of text. It may be above the first line or below the last line.

2. He finds the usage of 'punctuation' here odd, because it usually refers to characters entered into the text.

3. Why is single/double-sided setting and not how many interlinear punctuation marks between two lines of text affect the line gap (per the paragraph below)?

> To ensure appropriate positioning of interlinear punctuation, the line gap for single-sided setting shall not be smaller than half of the font-size, while the line gap for double-sided setting shall not be smaller than 5/8 (=1/2+1/8) of the font size

File: index.html [hunk 2]
```diff
@@ -2078,7 +2078,87 @@ <h4>
...
+          <p its-locale-filter-list="zh-hans" lang="zh-hans">无论单面装、双面装，也无论横排、直排，原则上行距应保持一致。这一点与 2.3.5.4 阐述内容一致。在最初设计基本版式时就必须提前考虑好行间标点的情况，并设置好足够大的行距。应尽量避免为了行间标点而强行局部撑大单行行距的做法。</p>
+          <p its-locale-filter-list="zh-hant" lang="zh-hant">無論單面裝、雙面裝，也無論橫排、直排，原則上行距應保持一致。這一點與 2.3.5.4 闡述內容一致。在最初設計基本版式時就必須提前考慮好行間標點的情況，並設置好足夠大的行距。應盡量避免為了行間標點而強行局部撐大單行行距的做法。</p>
+
+          <p its-locale-filter-list="en" lang="en">To ensure appropriate positioning of interlinear punctuation, the line gap for single-sided setting shall not be smaller than half of the font-size, while the line gap for double-sided setting shall not be smaller than 5/8 (=1/2+1/8) of the font size.</p>
```
Comment (xfq) [inline index.html]:
```suggestion
          <p its-locale-filter-list="en" lang="en">To ensure appropriate positioning of interlinear punctuation, the line gap for single-sided setting shall not be smaller than half of the font size, while the line gap for double-sided setting shall not be smaller than 5/8 (=1/2+1/8) of the font size.</p>
```

**Comment by huijing** [comment]
> I discussed this PR with @r12a offline. The following is a summary. He can explain in more detail.
> 
> 1. Interlinear line marks (行间线) refer to the line between two lines of text, but it is not necessarily between the two lines of text. It may be above the first line or below the last line.
> 2. He finds the usage of 'punctuation' here odd, because it usually refers to characters entered into the text.
> 3. Why is single/double-sided setting and not how many interlinear punctuation marks between two lines of text affect the line gap (per the paragraph below)?
> 
> > To ensure appropriate positioning of interlinear punctuation, the line gap for single-sided setting shall not be smaller than half of the font-size, while the line gap for double-sided setting shall not be smaller than 5/8 (=1/2+1/8) of the font size

@xfq, 第一点跟第三点不只是针对翻译而已吧？是不是中文版本也得改一改呢？
至于第二点，标点不翻译成 punctuation 那该用什么呢？

**Comment by r12a** [comment]
I don't know whether it's helpful, but just in case, here's how i describe these things in my own notes:

https://r12a.github.io/scripts/han/#baselines (third paragraph)
https://r12a.github.io/scripts/han/#emphasis

One other comment: the use of the word 'shall' indicates a prescriptive approach (like a style guide) rather than a descriptive approach.  I'd suggest to use 'will' in cases where you are indicating some future usage, or to drop the word altogether when describing normal behaviour.  To indicate expected/desirable outcomes (as in the quoted text in the comment above), i'd use the word 'should'.

hth

**Comment by xfq** [comment]
会议纪要：https://www.w3.org/2022/10/18-clreq-minutes.html#t03

-----

和Richard讨论之后，他有两点建议：

1. "The situation where both these punctuation are applied to **a run of text**" 听起来像「双面装」只是对**某一段文字**才有用，但实际上这是整本书的一个属性。
2. 他建议我们把shall改成must，因为shall带有很强的命令的感觉，而且是个较老的词，也过于正式，must没有这种感觉，也更加常用。

**Comment by huijing** [comment]
@xfq 我把以上的反馈以及之前讨论的都改了，麻烦你看看

File: index.html (hunk 1 above)
Comment (xfq) [reply index.html]: Fixed.

File: index.html [hunk 3]
```diff
@@ -2078,7 +2078,81 @@ <h4>
...
+      <p its-locale-filter-list="zh-hant" lang="zh-hant">傳統中文古籍中的「句讀」也都是「行間標點」。傳統中文古籍排版雖然已經超出了本文檔的範圍，但是行間標點的基本原則可參照本節內容。</p>
+
+      <p its-locale-filter-list="en" lang="en">In vertical typesetting, the interlinear lines are positioned to the left of the text, while the Emphasis Dot is positioned to the right of the text. The situation where both these marks are applied at the same time is known as double-sided setting. In horizontal typesetting, both the interlinear lines and Emphasis Dot are positioned to the bottom of the text. This situation is known as a single-side setting. In principle, text typeset horizontally will not use double-sided setting.</p>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">直排时，行间线标注于文字左侧，着重号标注于文字右侧。像这样，文字两侧均加排行间标点的排版方式叫「双面装」。横排时，行间线、着重号均标注于文字下方。像这样，文字单侧加排行间标点的排版方式叫「单面装」。原则上，横排书不使用双面装。</p>
+      <p its-locale-filter-list="zh-hant" lang="zh-hant">直排時，行間線標註於文字左側，著重號標註於文字右側。像這樣，文字兩側均加排行間標點的排版方式叫「雙面裝」。橫排時，行間線、著重號均標註於文字下方。像這樣，文字單側加排行間標點的排版方式叫「單面裝」。原則上，橫排書不使用雙面裝。</p>
+
+      <ol>
+        <li id="id203">
+          <h5>
```
Comment (xfq) [inline index.html]: If you look at the preview, there are some styling issues with the `h5` element.

We should probably turn it into a `p` element, or turn the `li` into a [section](https://respec.org/docs/#section).

File: index.html [hunk 4]
```diff
@@ -2078,7 +2078,81 @@ <h4>
...
+      </h4>
+
+      <p its-locale-filter-list="en" lang="en">Most punctuation marks in modern Chinese typesetting are interspersed between the text in a line, but the Proper Noun Mark (underline), the Book Title Mark (only for the type A wavy low line) and the Emphasis Dot need to be positioned between lines. These punctuation marks are known as “interlinear marks”. Of which, the Proper Noun Mark and the Book Title mark are known as interlinear lines.</p>
+      <p its-locale-filter-list="zh-hans" lang="zh-hans">现代中文排版的标点符号，大部分都随文字穿插在行内，但专名号（下划线）、书名号（仅指甲式的波浪线)、着重号要摆放在行与行之间，这些标点通称为「行间标点」。其中专名号（下划线）、书名号（仅指甲式的波浪线）合称为「行间线」。</p>
+      <p its-locale-filter-list="zh-hant" lang="zh-hant">現代中文排版的標點符號，大部分都隨文字穿插在行內，但專名號（下劃線）、書名號（僅指甲式的波浪線)、著重號要擺放在行與行之間，這些標點通稱為「行間標點」。其中專名號（下劃線）、書名號（僅指甲式的波浪線）合稱為「行間線」。</p>
 
+      <p its-locale-filter-list="en" lang="en">Jùdòu (句读) for ancient Chinese texts were all interlinear marks. Although the typesetting of ancient Chinese texts is beyond the scope of this document, the guidance provided in this section can still serve as a reference for the basic principles of interlinear marks.</p>
```
Comment (xfq) [inline index.html]: Maybe something like "Jùdòu (句读, old-style punctuation)"?

File: index.html (hunk 3 above)
Comment (huijing) [reply index.html]: Updated

File: index.html (hunk 4 above)
Comment (huijing) [reply index.html]: Updated

**Comment by xfq** [comment]
Merging per https://www.w3.org/2022/11/29-clreq-minutes.html#t06

**Comment by xfq** [comment]
Thank you for your hard work on this PR!

**Pull request merged**